kitsune/questions/views.py

```diff
@@ -1553,3 +1567,24 @@ def _add_tag(request, question_id):
 def _init_watch_form(request, event_type="solution"):
     initial = {"event_type": event_type}
     return WatchQuestionForm(request.user, initial=initial)
+
+
+def _add_to_moderation_queue(request, object):
+    object.is_spam = True
```

Comment:
The save operation should not be in this function but part of the view along with the rest options on the object.

Reply:
I have it here because the object needs to be saved before we save the `FlaggedObject` instance.